runtime-config.md

```diff
@@ -22,6 +23,13 @@ In Linux, for e.g., the network namespace could be configured in this hook.
 
 If a hook returns a non-zero exit code, then an error including the exit code and the stderr is returned to the caller and the container is torn down.
 
+### Post-start
+
+The post-start hooks are called after the user process is started.
+It can be used for notifying user that real process is spawned.
```

Comment:
There is nothing about "Sidecar application" as well as "sidecar process" in google, so seems like not so familiar language. cloudfoundry use-case isn't sidecar, but notfiy use-case indeed.